source/documentation/common/install/appe-Set_up_Cinderlib.adoc

```diff
@@ -20,15 +20,6 @@ endif::[]
 # subscription-manager repos --enable=openstack-16.2-cinderlib-for-rhel-8-x86_64-rpms
 ----
 
-ifdef::ovirt-doc[]
```

Comment:
dropepd as we provide yoga instead of ussuri and we provide it enabled by default.